### ISSUE TITLE
Issue with v-wave in Nuxt 3 with TypeScript

### ISSUE THREAD

**Issue by zalment** [justintaddei/v-wave#819]
### Library Versions:

```
nuxt: 3.13.2
vue: 3.5.4
v-wave: 3.0.2
vue-tsc: 2.1.6
```

When trying to use v-wave with TypeScript in a Nuxt 3 project, I'm receiving the following error:
```
ERROR(vue-tsc) Property 'directive' does not exist on type 'App<Element>'.
 FILE  ..../node_modules/v-wave/nuxt/runtime/plugin.ts:14:14

    12 |     },
    13 |   })
  > 14 |   app.vueApp.directive(${options.directive}-trigger, {
       |              ^^^^^^^^^
    15 |     ...vWaveTrigger,
    16 |     getSSRProps() {
    17 |       return {}

```

**Comment by justintaddei** [comment]
Could you share a code sandbox or a repo with the error? I've been unable to reproduce it.

**Comment by zalment** [comment]
Thanks for taking a look! After investigating further, I realized this wasn't actually an issue with v-wave itself. The error was caused by a TypeScript declaration I had in my project, specifically:

```
declare module 'vue' {
  interface ComponentCustomProperties {
    $breakpoint: BreakpointI;
    $iMask: IMaskI;
  }
}
```

coming from [here](https://github.com/nuxt/nuxt/issues/28707#issuecomment-2322297706)
This declaration, which I’d added as a fix for other components, unintentionally conflicted with v-wave. Removing this declaration (and upgrading to vue 3.5.12 for that issues) resolved the issue entirely.

Thanks again for your help, and sorry for any confusion! I'll go ahead and close this issue.
